code/game/machinery/doors/brigdoors.dm

```diff
@@ -30,7 +33,7 @@
 	var/activation_time = 0
 	var/timer_duration = 0
 
-	var/timing = FALSE // boolean, true/1 timer is on, false/0 means it's not timing
+	var/timing = TIMER_OFF		// boolean, true/1 timer is on, false/0 means it's not timing
```

Comment:
This doesn't need a define, just use TRUE/FALSE. Also, no alignment indentation.

Reply:
removed